bark/benchmark/BUILD

```diff
@@ -5,10 +5,17 @@ py_library(
     imports = ['../../../python'],
     deps = [
         "//bark/runtime:runtime",
+        ":benchmark_result"
```

Comment:
I think paths in respect to the root folder are better as this allows moving code more easily.